Firmware/VestibularH1/app_ios_and_regs.h

```diff
@@ -171,5 +181,11 @@ typedef struct
 #define B_VALVE1                           (1<<3)       // 
 #define B_PULSE_VALVE0                     (1<<4)       // 
 #define B_PULSE_VALVE1                     (1<<5)       // 
+#define GM_MCA_CONFIG                      7            // 
+#define MSK_MCA_OFF                        0            // 
+#define MSK_MCA_FLOW0_X                    1            // 
+#define MSK_MCA_FLOW0_Y                    2            // 
+#define MSK_MCA_FLOW1_X                    3            // 
+#define MSK_MCA_FLOW1_Y                    4            // 
```

Comment:
```suggestion
#define MSK_MCA_DELTAX0                     1            // 
#define MSK_MCA_DELTAY0                     2            // 
#define MSK_MCA_DELTAX1                     3            // 
#define MSK_MCA_DELTAY1                     4            // 
```

These should also probably be renamed for consistency with the proposed YAML changes.

Reply:
Fixed on commit 390dfb43d2dcae74173c8d5969e07bb8e622b710